src/main/kotlin/no/nav/familie/ef/sak/integration/PdlClient.kt

```diff
@@ -76,38 +76,40 @@ class PdlClient(val pdlConfig: PdlConfig,
         return feilsjekkOgReturnerData(pdlResponse)
     }
 
-    fun hentAktørId(personIdent: String): PdlHentIdenter {
+    fun hentAktørId(personIdent: String): PdlAktørId {
```

Comment:
Hvordan funker dette for de tre andre kallene. Må kanskje ta høyde for at de også kan være "null" ?

Reply:
De andre er av typen PdlBolkResponse og behandles annerledes (og den som returnerer string da, den er til testing tror jeg)

Reply:
Ikke helt enig;
```
{
    "errors": [
        {
            "message": "Ikke autentisert",
            "locations": [
                {
                    "line": 2,
                    "column": 5
                }
            ],
            "path": [
                "personBolk"
            ],
            "extensions": {
                "code": "unauthenticated",
                "classification": "ExecutionAborted"
            }
        }
    ],
    "data": null
}
```

Reply:
```
{
    "errors": [
        {
            "message": "Ikke autentisert",
            "locations": [
                {
                    "line": 2,
                    "column": 5
                }
            ],
            "path": [
                "personBolk"
            ],
            "extensions": {
                "code": "unauthenticated",
                "classification": "ExecutionAborted"
            }
        }
    ],
    "data": null
}
```

Dette er iallefall reellt case når jeg testet å hente ut en liste med barn

Reply:
I tillegg har du rett ang andre feilkoder;

```
{
    "data": {
        "personBolk": [
            {
                "code": "not_found",
                "ident": "1234567891",
                "person": null
            }
        ]
    }
}
```

Reply:
Men "person" er `null` her - tar vi høyde for det? 🤔 (sorry, men jeg har mistet litt oversikten over hva vi tar høyde for og ikke nå... 😂)